public/js/main_script.js

```diff
@@ -0,0 +1,108 @@
+const API_key = 'e0b8ee73-8e36-4d46-9651-177fdcb541a8';
```

Comment:
Во всех языках программирования, принято называть константы в верхнем регистре.
тут должно быть
```suggestion
const API_KEY = 'e0b8ee73-8e36-4d46-9651-177fdcb541a8';
```